pkg/apis/events/v1/conversion.go

```diff
@@ -43,7 +44,8 @@ func Convert_v1_Event_To_core_Event(in *v1.Event, out *k8s_api.Event, s conversi
 	return nil
 }
 
-func Convert_core_Event_To_v1_Event(in *k8s_api.Event, out *v1.Event, s conversion.Scope) error {
+// ConvertCoreEventToV1Event convert *k8s_api.Event to *v1.Event.
+func ConvertCoreEventToV1Event(in *k8s_api.Event, out *v1.Event, s conversion.Scope) error {
```

Comment:
I'm afraid this will fail to compile, `Convert_core_Event_To_v1_Event` is used by `zz_generated.conversion.go`.

Ref: https://github.com/kubernetes/kubernetes/blob/c1e5e6a5562bdd968977a36720f56362fbf4aa26/pkg/apis/events/v1/zz_generated.conversion.go#L138

Reply:
I thought it would be regenerated. What should I do? Closed this PR because it doesn't need to change with golint?